docs/outlook/mobile-log-appointments.md

```diff
@@ -0,0 +1,553 @@
+        <bt:String id="residLabel" DefaultValue="Log to Contoso CRM"/>
+      </bt:ShortStrings>
+      <bt:LongStrings>
+        <bt:String id="residTooltip" DefaultValue="Log notes to Contoso CRM for this appointment."/>
+      </bt:LongStrings>
+    </Resources>
+  </VersionOverrides>
+</VersionOverrides>
+```
+
+> [!TIP]
+> To learn more about manifests for Outlook add-ins, see [Outlook add-in manifests](manifests.md) and [Add support for add-in commands for Outlook Mobile](add-mobile-support.md).
+
+### Implement capturing appointment notes
+
+In this section, learn how your add-in can extract appointment details when the user selects the **Log** button.
+
+1. From the same quick start project, open the file **./src/commands/commands.js** in your code editor.
+
+1. Replace the entire content of the **commands.js** file with the following JavaScript.
+
+    ```js
+    // Office is ready.
+    Office.onReady(function () {
+        // Add any initialization code here.
+      }
+    );
+
+    function logCRMEvent(event) {
+      console.log(`Subject: ${Office.context.mailbox.item.subject}`);
+      Office.context.mailbox.item.body.getAsync(
+        "html",
+        { asyncContext: "This is passed to the callback" },
+        function callback(result) {
+          if (result.status === Office.AsyncResultStatus.Succeeded) {
+            event.completed({ allowEvent: true });
+          } else {
+            console.error("Failed to get body.");
+            event.completed({ allowEvent: false });
+          }
+        }
+      );
+    }
+
+    function getGlobal() {
+      return typeof self !== "undefined"
+        ? self
+        : typeof window !== "undefined"
+        ? window
+        : typeof global !== "undefined"
+        ? global
+        : undefined;
+    }
+
+    const g = getGlobal();
+    g.logCRMEvent = logCRMEvent;
+    ```
+
+### Implement viewing appointment notes
+
+The **Log** button can be toggled to display **View** by setting the **EventLogged** custom property reserved for this purpose. Then when the user the user selects the **View** button, they can look at their logged notes for this appointment.
+
+Add the following function to **./src/commands/commands.js**. This function sets the **EventLogged** custom property on the current appointment item.
+
+```js
+function updateCustomProperties(event) {
+  Office.context.mailbox.item.loadCustomPropertiesAsync(
+    function callback(customPropertiesResult) {
+      if (customPropertiesResult.status === Office.AsyncResultStatus.Succeeded) {
+        let customProperties = customPropertiesResult.value;
+        customProperties.set("EventLogged", true);
+        customProperties.saveAsync(
+          function callback(setSaveAsyncResult) {
+            if (setSaveAsyncResult.status === Office.AsyncResultStatus.Succeeded) {
+              console.log("EventLogged custom property saved successfully.");
```

Comment:
Here apart for the log, we should also call
`event.completed({ allowEvent: true });`
`event = undefined;`

Reply:
Added